modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -227,6 +259,183 @@ public void testGetObjectRelationshipERCFieldNameInOneToManyRelationshipFromRela
+			_objectDefinition1.getRESTContextPath(), Http.Method.POST);
+
+		Assert.assertEquals(
+			0,
+			jsonObject.getJSONObject(
+				"status"
+			).get(
+				"code"
+			));
+
+		String objectEntryId = jsonObject.getString("id");
+
+		jsonObject = HTTPTestUtil.invoke(
+			null,
+			StringBundler.concat(
+				_objectDefinition1.getRESTContextPath(), StringPool.SLASH,
+				objectEntryId, "?nestedFields=", _objectRelationship.getName()),
+			Http.Method.GET);
+
+		JSONArray nestedObjectEntriesJSONArray = jsonObject.getJSONArray(
+			_objectRelationship.getName());
+
+		Assert.assertEquals(
+			nestedCustomObjectEntryNumber,
+			nestedObjectEntriesJSONArray.length());
+
+		JSONObject nestedObjectEntryJSONObject1 =
+			(JSONObject)nestedObjectEntriesJSONArray.get(0);
+
+		JSONObject nestedObjectEntryJSONObject2 =
+			(JSONObject)nestedObjectEntriesJSONArray.get(1);
+
+		_assertObjectEntryField(
+			nestedObjectEntryJSONObject1, _OBJECT_FIELD_NAME_2,
+			_NEW_OBJECT_FIELD_VALUE_1);
+
+		_assertObjectEntryField(
+			nestedObjectEntryJSONObject2, _OBJECT_FIELD_NAME_2,
+			_NEW_OBJECT_FIELD_VALUE_2);
+	}
+
+	@Test
+	public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInManyToOneRelationship()
+		throws Exception {
+
+		_objectRelationship = ObjectRelationshipTestUtil.addObjectRelationship(
+			_objectDefinition1, _objectDefinition2, TestPropsValues.getUserId(),
+			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		JSONObject nestedObjectEntryStructureJSONObject = JSONUtil.put(
+			_objectRelationship.getName(),
+			JSONFactoryUtil.createJSONObject(
+				JSONUtil.put(
+					_OBJECT_FIELD_NAME_1, _NEW_OBJECT_FIELD_VALUE_1
+				).toString()));
+
+		JSONObject jsonObject = HTTPTestUtil.invoke(
+			nestedObjectEntryStructureJSONObject.toString(),
+			_objectDefinition2.getRESTContextPath(), Http.Method.POST);
+
+		Assert.assertEquals(
+			0,
+			jsonObject.getJSONObject(
+				"status"
+			).get(
+				"code"
+			));
+
+		String objectEntryId = jsonObject.getString("id");
+
+		jsonObject = HTTPTestUtil.invoke(
+			null,
+			StringBundler.concat(
+				_objectDefinition2.getRESTContextPath(), StringPool.SLASH,
+				objectEntryId, "?nestedFields=",
+				StringBundler.concat(
+					"r_", _objectRelationship.getName(), "_",
+					StringUtil.replaceLast(
+						_objectDefinition1.getPKObjectFieldName(), "Id", ""))),
+			Http.Method.GET);
+
+		JSONObject nestedObjectEntryJSONObject = jsonObject.getJSONObject(
+			StringBundler.concat(
+				"r_", _objectRelationship.getName(), "_",
+				StringUtil.replaceLast(
+					_objectDefinition1.getPKObjectFieldName(), "Id", "")));
+
+		Assert.assertNotNull(nestedObjectEntryJSONObject);
+
+		_assertObjectEntryField(
+			nestedObjectEntryJSONObject, _OBJECT_FIELD_NAME_1,
+			_NEW_OBJECT_FIELD_VALUE_1);
+	}
+
+	@Test
+	public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRelationship()
+		throws Exception {
+
+		int nestedCustomObjectEntryNumber = 2;
```

Comment:
This parameter is not necessary as you could calculate it from the nestedCustomObjectEntryFieldValues array size

Reply:
See https://github.com/liferay-headless/liferay-portal/pull/993/commits/871eb3550ddbd906676b3fd33bdd988f059d6cc3.